python/daqconf/apps/trigger_gen.py

```diff
@@ -339,11 +341,11 @@ def get_trigger_app(
                                                           window_time=10000,  # should match whatever makes TPSets, in principle
                                                           buffer_time=10*ticks_per_wall_clock_s//1000, # 10 wall-clock ms
                                                           activity_maker_config=temptypes.ActivityConf(ACTIVITY_CONFIG[j]))),
-                                DAQModule(name = f'tasettee_region_{region_id}_{plane}_{j}', plugin = "TASetTee")]
+                                DAQModule(name = f'tasettee_{region_id}_{plane}_{j}', plugin = "TASetTee")]
```

Comment:
This could be part of the same legend mentioned above, that the naming follows `region_plane_id`, since we are removing the region from the string name (which I agree is a good thing).